lib/crypto/src/pedersen/instance/starknet.rs

```diff
@@ -0,0 +1,205 @@
+//! This module contains the pedersen hash function parameters for Starknet.
+
+use crate::{
+    arithmetic::uint::U256,
+    curve::{
+        sw::{Affine, SWCurveConfig},
+        CurveConfig,
+    },
+    field::fp::{Fp256, FpParams, LIMBS_256},
+    fp_from_num, from_num,
+    pedersen::params::PedersenParams,
+};
+#[derive(Clone, Default, PartialEq, Eq)]
+/// Starknet's Curve Details.
+pub struct StarknetCurveConfig;
```

Comment:
```suggestion

/// Starknet's Curve Details.
#[derive(Clone, Default, PartialEq, Eq)]
pub struct StarknetCurveConfig;
```
nit: we usually place doc comments above attributes, same for other places in the file